src/app/core/data/metadatafield-parsing.service.ts

```diff
@@ -0,0 +1,19 @@
+import { DSpaceRESTv2Serializer } from '../dspace-rest-v2/dspace-rest-v2.serializer';
+import { DSpaceRESTV2Response } from '../dspace-rest-v2/dspace-rest-v2-response.model';
+import { RestRequest } from './request.models';
+import { ResponseParsingService } from './parsing.service';
+import { Injectable } from '@angular/core';
+import { MetadatafieldSuccessResponse, MetadataschemaSuccessResponse, RestResponse } from '../cache/response.models';
+import { MetadataField } from '../metadata/metadatafield.model';
+
+@Injectable()
+export class MetadatafieldParsingService implements ResponseParsingService {
```

Comment:
TypeDocs on this class please.  Also, the `parse` method below could use a brief description